.gitignore

```diff
@@ -3,4 +3,5 @@
 **/coverage
 **/.DS_Store
 .npmrc
+**/.env
```

Comment:
just curious -- why did we have to add this?

Reply:
Good question. We do not need to have this. I will remove it. 
FYI, The reason adding this is that I can to reduce the following warning message during build time by adding `GENERATE_SOURCEMAP=false` into the .env. 
```
Failed to parse source map from '/node_modules/eth-rpc-errors/src/utils.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/eth-rpc-errors/src/utils.ts'
```